server/api/status.go

```diff
@@ -39,11 +40,13 @@ func newStatusHandler(svr *server.Server, rd *render.Render) *statusHandler {
 // @Success  200  {object}  versioninfo.Status
 // @Router   /status [get]
```

Comment:
I am not sure whether it is suitable. ref https://github.com/tikv/pd/issues/8748#issuecomment-2440939642

Reply:
How about /member or /health?

Reply:
/members return all member info according etcd.ListEtcdMembers
/health is to query the status of each node through the /ping interface to return information about all the nodes, if we don't add a new api, we won't be able to get the result of whether the load region is complete or not.

Reply:
status is more like some fixed information to me, at least for the current definition.

Reply:
> put it under /status, which is appropriate from the name, but currently status returns compiled information.

Yes, as I said in the issue.